docs/index.md

```diff
@@ -1300,6 +1301,45 @@ class ExplicitFilePathSettings(BaseSettings):
         )
 ```
 
+### Azure Key Vault
+
+You must set two parameters:
+
+- `url`: For example, `https://my-resource.vault.azure.net/`.
+- `credential`: If you use `DefaultAzureCredential`, in local you can execute `az login` to get your identity credentials. The identity must have a role assignement (the recommended one is `Key Vault Secrets User`), so you can have access to the secrets.
+
+```
```

Comment:
``````suggestion
```py
``````

Reply:
I can't do this because there's no Key Vault resource to test against.